Kernel/Memory/MemoryManager.cpp

```diff
@@ -45,6 +45,14 @@ __attribute__((section(".super_pages"))) static u8 super_pages[4 * MiB];
 
 namespace Kernel::Memory {
 
+ErrorOr<FlatPtr> page_round_up(FlatPtr x)
+{
+    if (x > (explode_byte(0xFF) & ~0xFFF)) {
+        return Error::from_errno(EOVERFLOW);
+    }
```

Comment:
Seems like we should return `EINVAL` rather than `EOVERFLOW` since the error indicates of a bogus address argument.
Then we can use `TRY()` in various places below :)

Reply:
Sounds good, done! I think `MemoryManager.cpp`, `VirtualRange.cpp` and `mmap.cpp` were the only files that we can simplify with this, but please let me know if I'm missing anything.